include/common_functions

```diff
@@ -309,6 +314,13 @@ setup()
         setup_libsodium
         popd > /dev/null
     fi
+
+    if test ! -d "$libs"/micropython; then
+        git clone https://github.com/unikraft/lib-micropython "$libs"/micropython
+        pushd "$libs"/micropython > /dev/null
+        setup_micropython
+        popd > /dev/null
```

Comment:
Oh, yes these are not required, because they do nothing useful. Please remove them and the function `setup_micropython`. Sorry for the bad suggestion the first time.